File: integration_test_data/CppData/aplusb.test.cpp
#define PROBLEM "https://judge.yosupo.jp/problem/aplusb"
#include <iostream>
#include "macros.hpp"
#include "aplusb.hpp"
using namespace std;

int main()
{
    int a, b;
    cin >> a >> b;

    int c = aplusb(a, b);
    OUT(c);
    return 0;
}

File: integration_test_data/JavaData/examples/Aplusb_test.java
// competitive-verifier: PROBLEM https://judge.yosupo.jp/problem/aplusb
package examples;
import java.util.Scanner;
import examples.Aplusb;

public class Aplusb_test {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        System.out.println(Aplusb.plus(a, b));
    }
}
